<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormRequestDb</class>
 <widget class="QWidget" name="FormRequestDb">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <widget class="QTabWidget" name="tabW_data">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_database">
      <attribute name="title">
       <string>Данне с таблицы</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pb_update">
           <property name="text">
            <string>Обновить данные</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="tw_infoDb"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Выполнить запрос</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="vl_box_forSplitter">
         <item>
          <widget class="QGroupBox" name="gb_requestDb">
           <property name="title">
            <string>Запрос:</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QTextEdit" name="te_requestDb"/>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QGroupBox" name="groupBox">
                <property name="title">
                 <string/>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <item>
                  <widget class="QComboBox" name="cb_comSQL">
                   <item>
                    <property name="text">
                     <string>Выберете запрос</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>CREATE</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>SELECT</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>INSERT</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>UPDATE</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>DELETE</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pb_startRequestDb">
                <property name="text">
                 <string>Выполнить
 запрос</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="gb_answerDb">
           <property name="title">
            <string>Результат:</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QTextEdit" name="te_result"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
